tests/unit/policy_test.py

```diff
@@ -894,109 +894,80 @@ def test_surface_policy_moves_back_to_object(self):
                 # ------|------------------|-------------|----------------|-------------
-                #  1    | MoveForward      | True        | False          | dynamic_call
+                #  0    | None             | True        | False          | first_step
+                #  1    | MoveForward      | True        | False          | touch_object
                 #  2    | OrientHorizontal | True        | False          | dynamic_call
-                #  3    | OrientVertical   | False       | True           | dynamic_call
+                #  3    | OrientVertical   | KeyError    | True           | dynamic_call
                 #  4    | MoveTangentially | True        | False          | dynamic_call
                 #  5    | MoveForward      | True        | False          | dynamic_call
                 #  6    | OrientHorizontal | True        | False          | dynamic_call
-                #  7    | OrientVertical   | False       | True           | dynamic_call
+                #  7    | OrientVertical   | KeyError    | True           | dynamic_call
                 #  8    | MoveTangentially | True        | False          | dynamic_call
                 #  9    | MoveForward      | True        | False          | dynamic_call
                 #  10   | OrientHorizontal | True        | False          | dynamic_call
-                #  11   | OrientVertical   | False       | True           | dynamic_call
+                #  11   | OrientVertical   | KeyError    | True           | dynamic_call
                 #  12   | MoveTangentially | True        | False          | dynamic_call
                 # falls off object
-                #  13   | OrientHorizontal | True        | False          | touch_object
-                #  14   | OrientHorizontal | True        | False          | touch_object
-                #  15   | OrientHorizontal | True        | False          | touch_object
-                #  16   | OrientHorizontal | True        | False          | touch_object
-                #  17   | OrientHorizontal | True        | False          | touch_object
-                #  18   | OrientHorizontal | True        | False          | touch_object
-                #  19   | OrientHorizontal | True        | False          | touch_object
-                #  20   | OrientHorizontal | True        | False          | touch_object
-                #  21   | OrientHorizontal | True        | False          | touch_object
-                #  22   | OrientHorizontal | True        | False          | touch_object
-                #  23   | OrientHorizontal | True        | False          | touch_object
-                #  24   | OrientHorizontal | True        | False          | touch_object
-                #  25   | OrientVertical   | True        | False          | touch_object
+                # dataloder runs TouchObject positioning procedure in a tight loop only
+                # exposing a new observation at the end of the loop
```

Comment:
does this mean, finding the object is not counted as Monty steps anymore now?

Reply:
For others' benefit, we met and determined that finding the object is no longer counted as Monty's steps. I am still in the process of confirming that this is the cause of the observed benchmark differences.